src/local_storage.rs

```diff
@@ -13,20 +13,25 @@
 // limitations under the License.
 
 use std::default::Default;
-use std::collections::BTreeMap;
 
 use details::rpc::AnchoringRpcConfig;
 use details::btc;
 use observer::AnchoringObserverConfig;
 
+#[derive(Deserialize, Serialize, Clone, Debug, PartialEq)]
+pub struct AnchoringKeypair {
+    pub address: btc::Address,
+    pub private_key: btc::PrivateKey,
+}
+
 /// Private part of anchoring service configuration stored on a local machine.
 #[derive(Deserialize, Serialize, Clone, Debug, PartialEq)]
 pub struct AnchoringNodeConfig {
     /// Rpc configuration. Must exist if node is validator.
     /// Otherwise node can only check `lect` payload without any checks with `bitcoind`.
     pub rpc: Option<AnchoringRpcConfig>,
     /// Set of private keys for each anchoring address.
-    pub private_keys: BTreeMap<String, btc::PrivateKey>,
+    pub private_keys: Vec<AnchoringKeypair>,
```

Comment:
Can you write your own serialize and deserialize functions that would convert data between `Vec<AnchoringKeypair>` in toml and `BTreeMap<btc::Address, btc::PrivateKey>` in memory?